doc/configuration-v3_2_experimental.md

```diff
@@ -53,6 +53,7 @@ The Ignition configuration is a JSON document conforming to the following specif
       * **_guid_** (string): the GPT unique partition GUID.
       * **_wipePartitionEntry_** (boolean) if true, Ignition will clobber an existing partition if it does not match the config. If false (default), Ignition will fail instead.
       * **_shouldExist_** (boolean) whether or not the partition with the specified `number` should exist. If omitted, it defaults to true. If false Ignition will either delete the specified partition or fail, depending on `wipePartitionEntry`. If false `number` must be specified and non-zero and `label`, `start`, `size`, `guid`, and `typeGuid` must all be omitted.
+      * **_resizePartition_** (boolean) whether or not the existing partition should be resized. If omitted, it defaults to false. If true, Ignition will follow the `Partition Reuse Semantics` table, and if the `partition` is present, `shouldExist` is true, and `wipePartitionEntry` is false and the partition size doesn't match, then Ignition will resize the partition.
```

Comment:
"If true, Ignition will resize an existing partition if it matches the config in all respects except the partition size."?

The Partition Reuse Semantics table should also be updated, or at the least, a note should be added giving the semantics for the resize case.